Documentation/ApiOverview/Typo3CoreEngine/Database/Index.rst

```diff
@@ -193,6 +193,38 @@ Command keywords and values
          make a copy of the record on the same page.
 
 
+ - :Command:
+         copyToLanguage
+   :Type:
+         integer
+   :Value:
+         A new DataHandler command 'copyToLanguage' has been introduced. It behaves like 'localize' command
```

Comment:
I'd drop the first sentence since this is a reference and the command won't "have been introduced" for eternity. ;-)

Reply:
ok